lib/screens/events/events_repository.dart

```diff
@@ -52,3 +57,36 @@ class FakeEventsRepository implements EventsRepository {
     }
   }
 }
+
+class APIEventsRepository implements EventsRepository {
+  final String classTag = "APIEventsRepository";
+  @override
+  Future<List<dynamic>> events() async {
+    final String tag = classTag + "getEvents";
+    http.Response response;
+    try {
+      response = await sl.get<http.Client>().get(
+        S.getEventsUrl,
+        headers: <String, String>{
+          'accept': 'application/json',
```

Comment:
🟤 
reason for adding this? I feel it is not useful.

Reply:
It is working properly. What should I do?

Reply:
It should work w/o adding this header. If it does, remove it.

Reply:
Done